support/deployNextFromCI.ts

```diff
@@ -7,6 +7,7 @@
   const childProcess = await import("child_process");
   const { promisify } = await import("util");
   const exec = promisify(childProcess.exec);
+  const maxBuffer = 1024 * 2048;
```

Comment:
Nitpick: WDYT about `1024 * 1024 * <factor>`? I saw that format used in a few places to make it more clear what the adjusted value is. Up to you.